source/_integrations/renault.markdown

```diff
@@ -36,6 +36,25 @@ This integration provides the following platforms:
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+Locale:
+    description: "The country code used to connect to the Renault servers."
+    required: true
+    type: string
+Username:
+    description: "The username used to connect to the Renault servers."
+    required: true
+    type: string
+Password:
+    description: "The password used to connect to the Renault servers."
+    required: true
+    type: string
+Kamereon account id:
+    description: "The account id the vehicles are assigned to."
```

Comment:
```suggestion
    description: "The Kamereon account ID that your vehicles are assigned to. If there is only one account available it will be automatically selected."
```